java/arcs/core/host/AbstractArcHost.kt

```diff
@@ -391,21 +415,26 @@ abstract class AbstractArcHost(
             if (isRunning(arcId)) {
                 return@launch
             }
-
             val context = lookupOrCreateArcHostContext(arcId)
-            startArc(
-                Plan.Partition(
-                    arcId,
-                    hostId,
-                    context.particles.map { (_, particleContext) ->
-                        particleContext.planParticle
-                    }
-                )
-            )
+            val partition = contextToPartition(arcId, context)
```

Comment:
You probably don't need this right? if paused, them startArc() already adds to pausedArcs and returns?

Reply:
yes you are right.